core/src/storage/cache.py

```diff
@@ -72,6 +73,17 @@ def get(key: int) -> Any:
     return _caches[_active_session_id].get(key)
 
 
+def delete(key: int) -> None:
```

Comment:
not sure if we need this; based on usage, you could as well `cache.set(key, None)`

Reply:
I agree we don't need it, setting to `None` is indeed functionally equivalent. The reasons I chose this approach are that it just seems cleaner and that persistent storage also has set/get/delete.